Userland/Libraries/LibJS/Runtime/DataViewConstructor.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2021, Idan Horowitz <idan.horowitz@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJS/Runtime/DataView.h>
+#include <LibJS/Runtime/DataViewConstructor.h>
+#include <LibJS/Runtime/Error.h>
+#include <LibJS/Runtime/GlobalObject.h>
+
+namespace JS {
+
+DataViewConstructor::DataViewConstructor(GlobalObject& global_object)
+    : NativeFunction(vm().names.DataView, *global_object.function_prototype())
+{
+}
+
+void DataViewConstructor::initialize(GlobalObject& global_object)
+{
+    auto& vm = this->vm();
+    NativeFunction::initialize(global_object);
+    define_property(vm.names.prototype, global_object.data_view_prototype(), 0);
+    define_property(vm.names.length, Value(1), Attribute::Configurable);
+}
+
+DataViewConstructor::~DataViewConstructor()
+{
+}
+
+Value DataViewConstructor::call()
```

Comment:
Let's also add the comment here as both call and construct are part of the constructor

```suggestion
// 25.3.2.1 DataView ( buffer [ , byteOffset [ , byteLength ] ] ), https://tc39.es/ecma262/#sec-dataview-buffer-byteoffset-bytelength
Value DataViewConstructor::call()
```

Reply:
added